keps/sig-release/2572-release-cadence/README.md

```diff
@@ -0,0 +1,452 @@
+example, Golang upgrades. This has to be mitigated on a case-by-case basis, in
+the same way as it is being done right now.
+
+#### Feature graduation
+
+Research discovered that only 5% of Kubernetes features advanced from Alpha to
+GA in the minimum 3 releases. However, the same research showed that reminders
+from the Release Team played a critical role in advancement of more than 50% of
+features. With a longer release cycle, this reminder activity can be
+expected to slow down. As such, advancement will need to be mitigated by making
+sure that SIGs keep track of their feature enhancement in more detail.
+
+#### Unprepared Kubernetes Users
+
+Kubernetes effectively moved to a *3 release per year* cadence in 2020, starting 
+with the Kubernetes 1.19 release. At the start of the Kubernetes 1.21 release, 
+there was communication that a permanent cadence change was under consideration, 
+however this KEP was not submitted and approved within the Kubernetes 1.21 release 
+cycle, so there is a risk that downstream consumers may be unaware that such a 
+change was under consideration and could be caught by surprise by a cadence change. 
+Some downstream projects, such as Helm have already begun 
+[planning](https://github.com/helm/community/blob/main/hips/hip-0002.md#minor-releases) 
+for this change.
+
+To mitigate this risk, SIG-Release will perform the following actions:
+
+* Once this KEP has merged, an email will be sent to the [k/dev](https://groups.google.com/g/kubernetes-dev) list
+* A community meeting occurring during the 1.22 Release Cycle will be used to communicate the change
+* Early in the Kubernetes 1.22 Release cycle, a blog will be written and published to 
+https://kubernetes.io/blog/ that fully explains this change.
+* A tweet (linking to the blog) will sent from the k8scontributors twitter account
+
+#### Increased enhancements lifecycle
+
+With a 4 releases/year cadence, an enhancement could graduate from alpha to beta
+to GA in 9 months, with truly trivial features sometimes skipping beta. On the
+proposed 3 releases/year cadence, the best possible case is 12 months for a
+3-phase features and 8 months if skipping beta if the graduation rules will not
+change. These drawn out timelines may cause more features to skip beta or take
+more risks in advancing phases, even when not confident. The mitigation here is
+human vigilance and engineering discipline to hold the line and say "no" when
+appropriate.
+
+## Design Details
+
+### Schedule Policy
+
+The exact details of the release schedule are created by the team leads before
+the actual cycle begins. SIG Release only provides a lightweight policy, which
+is defined as:
+
+1. The first Kubernetes release of a year should start at the second or third
+   week of January to provide people more room after coming back from the
+   Christmas holidays.
+
+2. The last Kubernetes release of a year should be finished by the middle of
+   December.
```

Comment:
In regards to comments from @jberkus and @liggitt (https://github.com/kubernetes/enhancements/pull/2567#discussion_r616257381), I think that this should be defined more precisely. I think that choosing something like December 10th (as proposed by @liggitt) makes sense.